agent_runtime_profile/.claude/references/generation-modes.md

```
@@ -56,8 +56,10 @@ Step 7 视频
 
 ## 目录差异
 
+> 下面的目录树仅说明项目结构，session cwd 已在 `projects/{name}/`，**调用工具时使用相对 cwd 的路径**（如 `videos/`、`reference_videos/`），不可带 `projects/{name}/` 前缀。
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)